### ISSUE TITLE
inconsistent naming for enums

### ISSUE THREAD

**Issue by OneCyrus** [dotansimha/graphql-code-generator#1193]
**Describe the bug**
with version 0.16 the uppercase issue with enums is fixed. but there is something with the name of the enum itself that isn't consistent anymore.

this gets generated for the enum:
```
export enum CIState {
  NONE = "NONE",
  IN_DEVELOPMENT = "IN_DEVELOPMENT",
  IN_USE = "IN_USE",
  IN_REMOVAL = "IN_REMOVAL",
  REMOVED = "REMOVED"
}
```

and this is the type which references the enum. you'll notice that the enum CIState has a lowercase I in here. this breaks the reference as it is case sensitive. the actual graphql schema has an uppercase I (CIState as in the enum definition)

```
 export type Desired = {
    ...
    state: Maybe<CiState>;
  };
```

**To Reproduce**
don't really have a repro available as the source is cloud source.

**Expected behavior**
use the name of the graphql schema for naming in all cases.

**Schema/Documents**
can't porovide the schema

**Environment:**
 - OS: WIndows 10
 - Codegen: 0.16.0
 - Node: 8.9.1

**Additional context**
0.15.x used CiState (lower case I) in all places

**Comment by OneCyrus** [comment]
btw. naming configuration looks like this:
```
config: {
    namingConvention: {
      default: "change-case#pascalCase",
      enumValues: "change-case#upperCase"
    }
  },
```

is there something i can change?

**Comment by SoyYoRafa** [comment]
Seems like a "no-change-case" mode would solve this.

**Comment by kamilkisiela** [comment]
#1214

**Comment by dotansimha** [comment]
@OneCyrus fixed and merged, can you please try this alpha version? 

```
 - graphql-code-generator@0.17.0-alpha.bb76bd65
 - graphql-codegen-core@0.17.0-alpha.bb76bd65
 - codegen-templates-scripts@0.17.0-alpha.bb76bd65
 - graphql-codegen-plugin-handlebars-helpers@0.17.0-alpha.bb76bd65
 - graphql-codegen-plugin-helpers@0.17.0-alpha.bb76bd65
 - graphql-codegen-add@0.17.0-alpha.bb76bd65
 - graphql-codegen-flow-documents@0.17.0-alpha.bb76bd65
 - graphql-codegen-flow-resolvers@0.17.0-alpha.bb76bd65
 - graphql-codegen-flow@0.17.0-alpha.bb76bd65
 - graphql-codegen-fragment-matcher@0.17.0-alpha.bb76bd65
 - graphql-codegen-introspection@0.17.0-alpha.bb76bd65
 - graphql-codegen-schema-ast@0.17.0-alpha.bb76bd65
 - graphql-codegen-time@0.17.0-alpha.bb76bd65
 - graphql-codegen-typescript-apollo-angular@0.17.0-alpha.bb76bd65
 - graphql-codegen-typescript-client@0.17.0-alpha.bb76bd65
 - graphql-codegen-typescript-common@0.17.0-alpha.bb76bd65
 - graphql-codegen-typescript-graphql-files-modules@0.17.0-alpha.bb76bd65
 - graphql-codegen-typescript-mongodb@0.17.0-alpha.bb76bd65
 - graphql-codegen-typescript-react-apollo@0.17.0-alpha.bb76bd65
 - graphql-codegen-typescript-resolvers@0.17.0-alpha.bb76bd65
 - graphql-codegen-typescript-server@0.17.0-alpha.bb76bd65
 - graphql-codegen-typescript-stencil-apollo@0.17.0-alpha.bb76bd65
 - graphql-codegen-apollo-angular-template@0.17.0-alpha.bb76bd65
 - graphql-codegen-graphql-files-typescript-modules@0.17.0-alpha.bb76bd65
 - graphql-codegen-introspection-template@0.17.0-alpha.bb76bd65
 - graphql-codegen-typescript-mongodb-template@0.17.0-alpha.bb76bd65
 - graphql-codegen-typescript-react-apollo-template@0.17.0-alpha.bb76bd65
 - graphql-codegen-typescript-resolvers-template@0.17.0-alpha.bb76bd65
 - graphql-codegen-typescript-template@0.17.0-alpha.bb76bd65
```

**Comment by OneCyrus** [comment]
yes, that works now. thanks!

**Comment by gaberudy** [comment]
@SoyYoRafa you can ask for no naming changes like this:
```
generates:
  ./src/types/types.ts:
    plugins:
      - typescript-common
      - typescript-server
    config:
      avoidOptionals: true
      namingConvention:
        enumValues: keep
```

**Comment by dotansimha** [comment]
Fixed in 0.16.1 :)
